apps/mark-scan/backend/src/app.ui_strings.test.ts

```diff
@@ -1,26 +1,89 @@
+import os from 'os';
 import tmp from 'tmp';
 
-import { createMockUsb, runUiStringApiTests } from '@votingworks/backend';
+import {
+  MockUsb,
+  createMockUsb,
+  runUiStringApiTests,
+  runUiStringMachineConfigurationTests,
+} from '@votingworks/backend';
 import { fakeLogger } from '@votingworks/logging';
-
 import { buildMockInsertedSmartCardAuth } from '@votingworks/auth';
+import {
+  safeParseElectionDefinitionExtended,
+  testCdfBallotDefinition,
+} from '@votingworks/types';
+import {
+  BooleanEnvironmentVariableName,
+  getFeatureFlagMock,
+} from '@votingworks/utils';
+
+import {
+  fakeElectionManagerUser,
+  fakeSessionExpiresAt,
+} from '@votingworks/test-utils';
 import { Store } from './store';
 import { createWorkspace } from './util/workspace';
-import { buildApi } from './app';
+import { Api, buildApi } from './app';
+
+const mockFeatureFlagger = getFeatureFlagMock();
+
+jest.mock('@votingworks/utils', (): typeof import('@votingworks/utils') => {
+  return {
+    ...jest.requireActual('@votingworks/utils'),
+    isFeatureFlagEnabled: (flag) => mockFeatureFlagger.isEnabled(flag),
+  };
+});
 
 const store = Store.memoryStore();
 const workspace = createWorkspace(tmp.dirSync().name, { store });
+const mockAuth = buildMockInsertedSmartCardAuth();
+const electionPackage = safeParseElectionDefinitionExtended(
+  JSON.stringify(testCdfBallotDefinition)
+).unsafeUnwrap();
 
-afterAll(() => {
+afterEach(() => {
   workspace.reset();
 });
 
 runUiStringApiTests({
-  api: buildApi(
-    buildMockInsertedSmartCardAuth(),
-    createMockUsb().mock,
-    fakeLogger(),
-    workspace
-  ),
+  api: buildApi(mockAuth, createMockUsb().mock, fakeLogger(), workspace),
   store: store.getUiStringsStore(),
 });
+
+describe('configureMachine', () => {
```

Comment:
The most minor nit: This describe string `'configureMachine'` makes me think that there's a function/method named `configureMachine` that we're testing. But the method doesn't necessarily have that name, e.g. here it's `configureBallotPackageFromUsb`. `'machine configuration'` could work as a more general-purpose alternative

Reply:
Ah, good nit - updating to match the endpoint methods